server/sandbox_run_linux.go

```diff
@@ -869,10 +864,9 @@ func (s *Server) runPodSandbox(ctx context.Context, req *types.RunPodSandboxRequ
 			g.AddMount(proc)
 		}
 		rootPair := sandboxIDMappings.RootPair()
-		for _, path := range pathsToChown {
-			if err := os.Chown(path, rootPair.UID, rootPair.GID); err != nil {
-				return nil, fmt.Errorf("cannot chown %s to %d:%d: %w", path, rootPair.UID, rootPair.GID, err)
-			}
+		path := sbox.ResolvPath()
+		if err := os.Chown(path, rootPair.UID, rootPair.GID); err != nil {
```

Comment:
can you add a comment why we need a Chwon here but not for the others?

Reply:
it seems to not be required.  I've dropped it and simplified the patch further